### DISCUSSION TITLE
Seeders and Leechers Amount from external and Internet trackers

### DISCUSSION THREAD

**Discussion by BelieveInBunny** [torrust/torrust-index-archive#103] [Feature requests]
Hello,
It will be really good idea to add Seeders and Leechers Amount from external and Internet trackers for public trackers

it will make index look more alive and active.

![image](https://user-images.githubusercontent.com/50994966/168448342-04f94b55-d5c7-44ab-bfe8-d9eba6c735cb.png)

Thank you

**Comment by da2ce7** [comment]
Related to: https://github.com/torrust/torrust-index-archive/discussions/90

**Comment by da2ce7** [comment]
- https://github.com/torrust/torrust-index-gui/discussions/289
